foster55f/market2table-fe
# Vendor component

What does this PR do?
Creates VendorsContainer and VendorCard:

creates container for vendor cards connected to an individual market.
displays individual vendor cards and details

button that will display all these vendor components on MarketPage

button on VendorsContainer that will bring user back to specific market
 takes you back to page '/market/:id'

How to test?
Pull down branch
run 'npm start' and the site should open in browser via localhost:3000


Issues:
No issues

Relevant Screenshots:
![2020-02-21 13 32 12](https://user-images.githubusercontent.com/50148342/75072006-cdfcd700-54b3-11ea-8519-94d2b88ea8ad.gif)
